File: models/student.js

var mongoose = require("mongoose");
var Schema = mongoose.Schema;



var studentSchema = new Schema({

  name: {
    type: String,
    required: true
  },
  studentId: {
    type: String,
    required: true
  },
  password: {
    type: String,
    required: true,
    
  },
  
  
});

var student = mongoose.model("student", studentSchema);

module.exports = student;

File: MyApplication/app/src/main/java/com/example/myapplication/Student.java
package com.example.myapplication;

public class Student {




    private String name;

    private String studentId ;

    private String password ;



    public Student(String name, String studentId, String password) {

        this.name = name;
        this.studentId = studentId;
        this.password = password;

    }



    public String getName() {
        return name;
    }

    public String getStudentId() {
        return studentId;
    }

    public String getPassword() {
        return password;
    }


}

